include/ListView/ListViewDisplay.php

```diff
@@ -414,7 +416,8 @@ protected function buildActionsLink($id = 'actions_link', $location = 'top')
 
 
             if (
```

Comment:
@jack7anderson7  Here we should also stop relying on `& !$this->show_action_dropdown_as_delete` for the reason metioned in https://github.com/salesagility/SuiteCRM/pull/9420/files#r785936264